test/AppNotificationBuilderTests/APITests.cpp

```diff
@@ -347,86 +384,86 @@ namespace Test::AppNotification::Builder
 
             VERIFY_ARE_EQUAL(builder.BuildNotification().Payload(), expected);
         }
 
         TEST_METHOD(AppNotificationBuilderSetAudioWithSoundEventAndDuration)
         {
-            auto builder{ AppNotificationBuilder()
-                    .SetDuration(AppNotificationDuration::Long)
-                    .SetAudioEvent(AppNotificationSoundEvent::Reminder, AppNotificationAudioLooping::Loop) };
+            auto builder{ winrt::AppNotificationBuilder()
+                    .SetDuration(winrt::AppNotificationDuration::Long)
+                    .SetAudioEvent(winrt::AppNotificationSoundEvent::Reminder, winrt::AppNotificationAudioLooping::Loop) };
             auto expected{ L"<toast duration='long'><visual><binding template='ToastGeneric'></binding></visual><audio src='ms-winsoundevent:Notification.Reminder' loop='true'/></toast>" };
 
             VERIFY_ARE_EQUAL(builder.BuildNotification().Payload(), expected);
         }
 
         TEST_METHOD(AppNotificationBuilderMuteAudio)
         {
-            auto builder{ AppNotificationBuilder().MuteAudio() };
+            auto builder{ winrt::AppNotificationBuilder().MuteAudio() };
             auto expected{ L"<toast><visual><binding template='ToastGeneric'></binding></visual><audio silent='true'/></toast>" };
 
             VERIFY_ARE_EQUAL(builder.BuildNotification().Payload(), expected);
         }
 
         TEST_METHOD(AppNotificationBuilderBuildNotificationWithTooLargePayload)
         {
-            VERIFY_THROWS_HR(AppNotificationBuilder()
+            VERIFY_THROWS_HR(winrt::AppNotificationBuilder()
                 .AddText(std::wstring(5120, 'A').c_str())
                 .BuildNotification(), E_INVALIDARG);
         }
 
         TEST_METHOD(AppNotificationAddProgressBar)
         {
-            auto builder{ AppNotificationBuilder()
+            auto builder{ winrt::AppNotificationBuilder()
                 .AddText(L"Downloading this week's new music...")
-                .AddProgressBar(AppNotificationProgressBar()
+                .AddProgressBar(winrt::AppNotificationProgressBar()
                     .BindTitle()
                     .BindValueStringOverride()) };
-            auto expected{ L"<toast><visual><binding template='ToastGeneric'><text>Downloading this week's new music...</text><progress title='{progressTitle}' status='{progressStatus}' value='{progressValue}' valueStringOverride='{progressValueString}'/></binding></visual></toast>" };
-
+            auto expected{ L"<toast><visual><binding template='ToastGeneric'><text>Downloading this week&apos;s new music...</text><progress title='{progressTitle}' status='{progressStatus}' value='{progressValue}' valueStringOverride='{progressValueString}'/></binding></visual></toast>" };
+            WEX::Logging::Log::Comment(builder.BuildNotification().Payload().c_str());
```

Comment:
Is this a left over from a debugging session?

Reply:
Yup. I'm still working on unit tests but will remove when finished.